plugins/GSdx/GSRendererHW.cpp

```diff
@@ -53,7 +53,7 @@ void GSRendererHW::SetScaling() {
 	
 	//Only increase the buffer size, don't make it smaller, it breaks games (GH3)
 
-	if (!m_nativeres && m_width < (m_buffer_size * m_upscale_multiplier)){
+	if (m_upscale_multiplier !=1 && m_width < (m_buffer_size * m_upscale_multiplier)){
```

Comment:
I suspect the original line was wrong here.
It should probably have been
`if ((m_nativeres || m_upscale_multiplier != 1) &&  m_width < (m_buffer_size * m_upscale_multiplier)) {`
That way the size for custom res won't change and the size for native and scaling won't shrink.

It would also fix the custom res width < native res width black screen bug mentioned on the forums - it would prevent the texture cache targets being dropped every vsync. Since custom res is still in the code, it might as well be made to work as badly as it did before.